components/date-picker/picker.component.ts

```diff
@@ -406,14 +406,14 @@ export class NzPickerComponent implements OnInit, AfterViewInit, OnChanges, OnDe
     return this.dateHelper.format(value && (value as CandyDate).nativeDate, this.format);
   }
 
-  onInputKeyup(event: Event, isEnter: boolean = false): void {
-    if (isEnter && !this.realOpenState) {
+  onInputKeyup(event: Event, OpenPanel: boolean = false, emitValue: boolean = false): void {
+    if (OpenPanel && !this.realOpenState) {
```

Comment:
```suggestion
    if (!this.realOpenState) {
```
It seems only needs to change here, without adding a parameter.

Reply:
Thanks for your advice!